main.py

```diff
@@ -1,26 +1,36 @@
 from game import Game
 from board import Board
-from player import Player
+from player import Player, ComputerPlayer
 from validation import Validator
 from players import Players
 from console import Console
 
 
-
-def main():
-    player_1 = Player("1", "x")
-    player_2 = Player("2", "o")
-    players = Players(player_1, player_2)
-    validator = Validator()
-    console = Console()
-
+def setup_game(console, validator):
     console.print_greeting()
     console.print_instructions()
-    board_size = console.select_board_size(
-        "Let's build a board! Pick an odd number from 1 to 7: ", validator)
+    choice = console.selector(
+        "Let's pick your opponent!"
+        "\nPress 1 for Human or press 2 for Computer: ", validator.validate_choice)
+    if choice == 1:
+        players = Players(Player("1", "x"), Player("2", "o"))
+    else:
+        players = Players(ComputerPlayer("1", "x"), Player("2", "o"))
+
+    board_size = console.selector(
+        "Let's build a board! Pick an odd number from 1 to 7: ", validator.validate_size)
     board = Board(board_size)
```

Comment:
I see here you have logic in main (if, else). I wonder if you can put that elsewhere so you can test drive it.